com.unity.multiplayer.mlapi/Editor/MLAPIProfilerModule.cs

```diff
@@ -33,59 +35,61 @@ private class MLAPIProfilerCounter
         private class MLAPIProfilerModuleData
         {
             // Note: These fields are named this way for internal serialization
-            public List<MLAPIProfilerCounter> m_ChartCounters = new List<MLAPIProfilerCounter>();
-            public List<MLAPIProfilerCounter> m_DetailCounters = new List<MLAPIProfilerCounter>();
-            public string m_Name;
+            public List<MLAPIProfilerCounter> ChartCounters = new List<MLAPIProfilerCounter>();
+            public List<MLAPIProfilerCounter> DetailCounters = new List<MLAPIProfilerCounter>();
+            public string Name;
```

Comment:
@becksebenius-unity @kvassall-unity

Reply:
No. This cant be done. as the comment says above it. the fields are name that way for internal serialization